samples/FluentMigrator.Example.Migrations/FluentMigrator.Example.Migrations.csproj

```diff
@@ -2,11 +2,19 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Assembly containing migrations for the example project</Description>
+    <!-- By default, *.cs files will be automatically included as compile items,
+    but *.sql files will be automatically included as None items.  This disables that behavior. -->
```

Comment:
In this comment, it seems unclear whether "that behavior" that is disabled includes including *.cs files as compile items (which it doesn't).  So I'd replace "that behavior" with "automatically including None items", or similar.